apps/docs-snippets/src/guide/provider/provider.test.ts

```diff
@@ -0,0 +1,36 @@
+import { FUEL_NETWORK_URL, Provider, sleep } from 'fuels';
+
+async function fetchSomeExternalCredentials() {
+  return Promise.resolve('credential');
+}
+
+/**
+ * @group node
+ */
+describe('Provider', () => {
+  it('can be given options', async () => {
+    // #region provider-options
+    await Provider.create(FUEL_NETWORK_URL, {
+      fetch: async (url, requestInit) => {
```

Comment:
Can we add the argument's type notation here?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1822/commits/849a9ca1f42f33fa592f2f7ba51b6481dd850690